src/master/grafana/dashboards/InfiniBand/network-counters.json

```diff
@@ -176,7 +185,7 @@
         "alertThreshold": true
       },
       "percentage": false,
-      "pluginVersion": "7.5.7",
+      "pluginVersion": "8.5.0",
```

Comment:
whats the relevance of plugin version?

Reply:
This is something created by Grafana. I created the dashboard on the portal and exported it. Release notes: https://grafana.com/docs/grafana/latest/release-notes/release-notes-8-5-0/